crates/swc_ecma_minifier/tests/terser/compress/functions/issue_2630_3/output.js

```diff
@@ -1,12 +1,6 @@
-var x = 2,
-    a = 1;
-(function () {
-    (function f1(a) {
-        f2();
-        --x >= 0 && f1({});
-    })(a++);
-    function f2() {
-        a++;
-    }
-})();
```

Comment:
As I mentioned before, terser cannot inline into conflict variable names, great work for swc!